app/controllers/steps/details/start_controller.rb

```diff
@@ -1,7 +1,7 @@
 module Steps::Details
   class StartController < Steps::DetailsStepController
     def show
-      @can_start = current_tribunal_case&.cost_task_completed?
+      @can_start = current_tribunal_case.present?
```

Comment:
Indeed, we can chuck the start controller entirely. But happy to do that as a separate PR.